dev/com.ibm.ws.jpa_eclipselink_fat/bnd.bnd

```diff
@@ -0,0 +1,37 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.target: 1.8
+javac.source: 1.8
+
+src: \
+	fat/src,\
+	test-applications/helpers/DatabaseManagement/src,\
+	test-applications/example/src,\
+	test-framework/src
+
+fat.project: true
+tested.features: jdbc-4.0, beanvalidation-1.1, jpa-2.0, el-3.0
```

Comment:
Discussed this and it seems removing this might be in order

Reply:
Removed.